templates/blrules.erb

```diff
@@ -7,9 +7,13 @@
 # information.
 #
 ###############################################################################
+<% if @whitelists -%>
```

Comment:
We could just default them to empty, then we would not need that.

Reply:
Sorry, I’m not understand how it’s work. If whitelist not empty? Right now I am using the whitelist as a list of the protected perimeter networks. If the list is empty, then we do nothing.

Reply:
If we make sure that the whitelist parameter must always be an array, then we do not need the wrap with the if that checks if whitelists is set.

If the array is empty, we won't set any whitelist anyways.

This can be achieved with Puppet Data types.

Reply:
You are right! Reverted.